docs/NETWORKING.md

```diff
@@ -93,8 +99,9 @@ Application then references UUIDs of network instances to which it should be con
 
 ### Uplink
 
-Uplink denotes network adapter that a network instance is connected to for external connectivity.
-Network instance can be air-gapped, i.e. without uplink, thus not providing external connectivity.
+Uplink denotes network adapter used for EVE management, i.e. providing connectivity
+with the controller. Multiple adapters can be used for management and EVE will automatically
+fail-over between them when some loose connectivity.
```

Comment:
probably should be 'lost connectivity', loose seems to be not connected very tight.